t/unit/test_connection.py

```diff
@@ -54,6 +54,11 @@ def test_as_uri_when_mongodb(self):
         x = Connection('mongodb://localhost')
         assert x.as_uri()
 
+    @skip.unless_module('pymongo')
+    def test_as_uri_when_mongodb_srv(self):
+        x = Connection('mongodb+srv://localhost')
+        assert x.as_uri()
```

Comment:
Shouldn't this assert that the uri is `mongodb+srv://localhost`? Just asserting it doesn't return `Falsey` doesn't seem like a great test.

But I see that the other mongodb test does the same thing...